### ISSUE TITLE
use memfs to input/output "in-memory" files to ffmpeg

### ISSUE THREAD

**Issue by djardine** [fluent-ffmpeg/node-fluent-ffmpeg#896]
hi, I was wondering if anyone has had any luck with using an in-memory file system such as memfs as input and/or output file with fluent/ffmpeg. I'm able to use memfs createReadStream/createWriteStream with fluent, but I'd actually prefer not to use streams. Generating an output stream using mp4 container and the following flags caused playback issues in some browsers (firefox/edge) and players (they don't seem to be able to pick up the video length accurately):

`'-movflags frag_keyframe+faststart'`

Possibly I'm misusing the above flags, but that's what I found in the fluent FAQ.

Ideally I like to pass in a normal file path string into fluent .input() and.output() calls but have them reference in-memory file paths rather than std file system file paths, but I haven't been able to figure out a way to do that.

Thanks.

**Comment by tommyc38** [comment]
I wonder if this is my issue with GAE standard environment.  Everything works fine when ran on my local dev machine but won't in the cloud.

**Comment by FedericoCarboni** [comment]
@djardine No, it can't be done, if it is in-memory then it can only be passed to FFmpeg as a stream.
